### ISSUE TITLE
Realmente necessário listar os devs?

### ISSUE THREAD

**Issue by onhernandes** [woliveiras/nodejs-in-brazil#7]
É realmente necessário listar os devs que usam NodeJS no Brasil? Acredito que podemos focar o repo apenas em projetos e empresas, que basicamente são a mesma coisa em linhas gerais, já devs em si é um pouco fora do escopo, sugiro um repo a parte sobre isso, talvez. O que acham?

**Comment by woliveiras** [comment]
O legal de listarmos devs é centralizar "tudo" de Nodejs Brasileiro em um repositório só.

A ideia dessa lista surgiu por que eu estava procurando devs e empresas usando Nodejs no Google. Ai pensei em centralizar tudo em um local só.

**Comment by onhernandes** [comment]
@woliveiras e se deixarmos esse repo só pra projetos no geral e criarmos um segundo, ou melhor, um arquivo a parte listando os devs? Acredito que deixar o README só com os projetos já vai ficar muuuuito grande daqui algum tempo.

**Comment by IgorAntun** [comment]
Concordo com o @onhernandes. Vejo o valor em centralizar uma lista de devs brasileiros, mas realmente com o tempo fica inviável manter todos no README.
Pelo menos um arquivo DEVS.md poderia ser criado agora, e, caso a lista de projetos fique muito grande, criar um arquivo pra ela também.

**Comment by woliveiras** [comment]
Vou fazer essa alteração então, galera
